codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/SymbolVisitor.java

```diff
@@ -331,13 +338,32 @@ public Symbol stringShape(StringShape shape) {
     @Override
     public Symbol structureShape(StructureShape shape) {
         String name = getDefaultShapeName(shape);
+        if (shape.hasTrait(SyntheticClone.ID)) {
+            Optional<String> boundOperationName = getNameOfBoundOperation(shape);
+            if (boundOperationName.isPresent()) {
+                return SymbolUtils.createPointableSymbolBuilder(shape, name, rootModuleName)
+                        .definitionFile("./api_Op_" + boundOperationName.get())
+                        .build();
+            }
+        }
+
         Symbol.Builder builder = SymbolUtils.createPointableSymbolBuilder(shape, name, typesPackageName);
         if (shape.hasTrait(ErrorTrait.ID)) {
-            builder.definitionFile("./types/errors.go");
```

Comment:
Is there a reason for swapping this to include repeated code?

Reply:
That's an aspect of it having been part of the great namespace reconstruction of 2020. It doesn't really matter to me either way though, since it's just a build and return that's duped